### PR TITLE
Overwrite the upstream lerna 4 dep to 2.11.0

### PR DESCRIPTION
**Pull request by azatsarynnyy**
Signed-off-by: Artem Zatsarynnyi <azatsary@redhat.com>

### What does this PR do?
Overwrites the upstream lerna 4.0.0 dep to 2.11.0.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)
- [ ] Optional Companion PR for updating the HappyPath tests is approved and ready to be merged

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### Happy Path Channel

HAPPY_PATH_CHANNEL=stable

### CODE REVIEW

**Review by nickboldt — APPROVED** [review]

**Review by vzhukovs — APPROVED** [review]

**Review by benoitf — APPROVED** [review]

**Review by benoitf — APPROVED** [review]

File: dockerfiles/theia/Dockerfile [hunk 1]
```diff
@@ -61,8 +61,10 @@ RUN rm -rf ${HOME}/theia-source-code/examples/browser && \
     rm -rf ${HOME}/theia-source-code/examples/api-samples && \
     rm -rf ${HOME}/theia-source-code/examples/api-tests && \
     rm -rf ${HOME}/theia-source-code/packages/git && \
+    # ovewrite upstream's lerna 4.0.0 as Che-Theia is not adapted to it
+    sed -r -e "s/\"lerna\": \"..*\"/\"lerna\": \"2.2.0\"/" ${HOME}/theia-source-code/package.json && \
```
Comment (RomanNikitenko) [inline dockerfiles/theia/Dockerfile:65]: 2.11.0?
https://github.com/eclipse-che/che-theia/pull/1285/files

**Comment by RomanNikitenko** [comment]
`CI / docker-build` failed
the errors are the same as for a build when lerna 4 is used

**Comment by RomanNikitenko** [comment]
I checked that with the PR change lerna 4 is still used in package.json file

Screenshot 2022-01-19 at 12 33 15

**Comment by azatsarynnyy** [comment]
There was a missing `-i` to overwrite `package.json`.
Updated, checking the `docker-build` jobs.

**Review by RomanNikitenko — APPROVED** [review]

**Comment by nickboldt** [comment]
Cherrypicked to 7.42.x branch as c96c47de

**Pull request merged**